### DISCUSSION TITLE
Where is the configuration file

### DISCUSSION THREAD

**Discussion by aubreyz** [lukevella/rallly#1119] [Q&A]
This may be ignorance of Docker (my first docker container) - but where exactly is the configuration file in the docker config area. I want to change the email address set up on the docker install, and also to add the ALLOWED_EMAILS= option, but I cannot work out where this is stored. I don't want to have to reinstall the container from scratch just to change these parameters which must be somewhere.... 

Any help appreciated.

**Comment by gilgongo** [comment]
If I understand you correctly, the config file is `config.env` in wherever you put your rallly files (along with `docker-compose.yml` etc.)

I think you need run `docker compose restart` (from the above directory) after making any changes to `config.env` otherwise they won't take effect, but I'm not sure. The docs seem to be silent on that.

**Comment by aubreyz** [comment]
Yes apologies I did resolve. I have to fix the settings in the docker compose file, and then re-initialise the container.
